pkg/drivers/vfkit/driver_darwin.go

```diff
@@ -466,10 +468,29 @@ func (d *Driver) sendSignal(s syscall.Signal) error {
 	return proc.SendSignal(s)
 }
 
+var curVersionOnce = func() (string, error) { return syscall.Sysctl("kern.osproductversion") }
+
 func (d *Driver) supportsVirtiofs() bool {
-	macosVersion, _, err := crcos.RunWithDefaultLocale("sw_vers", "-productVersion")
+	supportsVirtioFS, err := macosNewerThan(curVersionOnce, "11.0.0")
```

Comment:
I don' think `macosNewerThan` is the right API here, `macosAtLeast` should be more correct.
macosNewerThan("11.0.0") means that "11.0.0" is to be excluded, which is not obvious/natural to me, I need to think twice about it. And my understanding is that `11.0.1` would pass this check, which is incorrect, the file sharing API is macOS12+